src/event/event.controller.ts

```diff
@@ -110,6 +126,8 @@ export class EventController {
     return this.eventService.attendEvent(slug, user.id, createEventAttendeeDto);
   }
 
+  // @Permissions(EventAttendeePermission.CancelEvent)
```

Comment:
This one looks like a puzzled place same as for `attendEvent` method.

There should be a top level guard for `UserPermission.AttendEvents` enum